README.md

```diff
@@ -229,6 +229,7 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [Stripe](https://github.com/ServiceStack/Stripe) - Typed .NET clients for stripe.com REST APIs.
 
 ### Exceptions
```

Comment:
Please resolve conflict

Reply:
@ngohungphuc Seems that the same one was added a few days ago: https://github.com/thangchung/awesome-dotnet-core/commit/095ac7275d1daf85fd1a5b6b51539cad80f0515a
So I think we can just close this PR?

Reply:
thank you